etna/core/mixins.py

```diff
@@ -230,7 +230,7 @@ def _save_metadata(self, archive: zipfile.ZipFile):
             output_file.write(metadata_bytes)
 
     def _save_state(self, archive: zipfile.ZipFile):
-        with archive.open("object.pkl", "w") as output_file:
+        with archive.open("object.pkl", "w", force_zip64=True) as output_file:
```

Comment:
Does it affects loading somehow? Do we have test for it?

Reply:
It doesn't affect loading, we have tests on saving/loading (not large files). Large files I tested manually.
As I understand the problem that with `archive.open` is that is should know about the size of the file to create a correct metainfo. We provide it with `force_zip64` to make sure it creates metainfo for large file.

Reply:
https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.open